model_compression_toolkit/ptq/keras/quantization_facade.py

```diff
@@ -140,15 +142,20 @@ def keras_post_training_quantization(in_model: Model,
                                                target_resource_utilization=target_resource_utilization,
                                                tb_w=tb_w)
 
-        tg = ptq_runner(tg, representative_data_gen, core_config, fw_info, fw_impl, tb_w)
+        float_graph = copy.deepcopy(tg)
```

Comment:
But this is not the float graph, it's the graph that is outputted from the core runner..
What am I missing?

(relevant to all facades so I'm not writing this again)